cluster/gce/gci/configure.sh

```diff
@@ -436,6 +443,56 @@ function install-docker {
   rm -rf /var/lib/apt/lists/*
 }
 
+# If we are on ubuntu we can try to install containerd
+function install-containerd {
+  # bailout if we are not on ubuntu
+  if ! command -v apt-get >/dev/null 2>&1; then
```

Comment:
if we get here, it means UBUNTU_INSTALL_CONTAINERD_VERSION or UBUNTU_INSTALL_RUNC_VERSION are set... should we error in non-ubuntu/non-amd64 envs? it means someone is running this requesting those be installed, and they aren't

Reply:
i can add a explicit check for ubuntu